Source/NSWindow.m

```diff
@@ -2701,6 +2705,50 @@ - (NSScreen *) screen
   return _screen;
 }
 
+- (void) applicationDidChangeScreenParameters: (NSNotification *)aNotif
+{
+  NSRect oldScreenFrame = [_screen frame];
+  int    screenNumber = [_screen screenNumber];
+  NSRect newScreenFrame;
+  NSRect newFrame;
+
+  // NSLog(@"[NSWindow - %@] Old/New Screen RC: %lu/%lu",
```

Comment:
Could you please remove these commented out lines or turn them into NSDebugLog statements?

Reply:
Done.